### DISCUSSION TITLE
[Cloud Engineer] Optimize geographic placement of workloads for user locations

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/patterns#33] [Ideas]
https://docs.aws.amazon.com/wellarchitected/latest/sustainability-pillar/optimize-geographic-placement-of-workloads-for-user-locations.html

Intent: Analyze network access patterns to identify where your customers are connecting from geographically. Select Regions and services that reduce the distance that network traffic must travel to decrease the total network resources required to support your workload.

**Comment by holanita** [comment]
[Issue created](https://github.com/Green-Software-Foundation/green-software-patterns/issues/65)
